src/metrics/distances.ts

```diff
@@ -1,13 +1,13 @@
-import { Piece, PrettyMove, UASymbol } from '../../cjsmin/src/chess';
+import { Piece, PrettyMove, UASymbol } from '../cjsmin/src/chess';
 import { UAPMap } from '../types';
 import { createUAPMap } from '../utils';
 import { Metric } from './metric';
 
 export class MoveDistanceMetric implements Metric {
-  distanceMap: UAPMap<{ 
+  distanceMap: UAPMap<{
     distance: number;
   }>;
-  avgDistanceMap: UAPMap<{ 
+  avgDistanceMap: UAPMap<{
```

Comment:
a common lint standard would resolve all this stuff

Reply:
most of these chagnes are bc of linting diffs

Reply:
any you prefer? ive got prettier on

Reply:
likewise but the workspace settins are different probabley